### PR TITLE
Add "Beyond #100000: You're breathtaking!"

### PR DESCRIPTION
**Pull request by adamscott**
Add a copy of godotengine/godot#100000 to the blog, including some CSS fixes. 

<details markdown=1>

Screenshot 2024-12-04 at 13-08-23 Beyond #100000 You're breathtaking! – Godot Engine webp

</details>

### CODE REVIEW

**Comment by hpvb** [comment]
This is great! Can we add the images etc to our own hosting? I'm not sure if github user content links are entirely stable.

**Comment by coppolaemilio** [comment]
I don't think we can use Keanu's picture here, I would replace it with something else. The share image generated by Github, or a picture of the repository @adamscott

**Comment by adamscott** [comment]
> This is great! Can we add the images etc to our own hosting? I'm not sure if github user content links are entirely stable.

Done! I reduced the crap out of the gifs and the pngs also.

> I don't think we can use Keanu's picture here, I would replace it with something else. The share image generated by Github, or a picture of the repository @adamscott

I'll do.

**Comment by adamscott** [comment]
Replaced the real photo by this:

beyond-100000-you-re-breathaking webp

(I'm the author of the sketch)

**Review by Chubercik — APPROVED** [review]

File: collections/_article/beyond-100000-you-re-breathaking.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Beyond #100000: You're breathtaking!"
+excerpt: "."
+categories: ["news"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/beyond-100000-you-re-breathaking.webp
+image_caption_title: "\"You're breathtaking!\""
+image_caption_description: "Sketch of Keanu Reeves at the 2019 #XboxE3 event"
```
Comment (akien-mga) [inline collections/_article/beyond-100000-you-re-breathaking.md]: Are you really telling me this was 5 whole years ago?! :older_man:

File: collections/_article/beyond-100000-you-re-breathaking.md [hunk 2]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Beyond #100000: You're breathtaking!"
+excerpt: "."
+categories: ["news"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/beyond-100000-you-re-breathaking.webp
```
Comment (akien-mga) [inline collections/_article/beyond-100000-you-re-breathaking.md]: Image name should be fixed accordingly.
```suggestion
image: /storage/blog/covers/beyond-100000-you-re-breathtaking.webp
```

File: collections/_article/beyond-100000-you-re-breathaking.md [hunk 3]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Beyond #100000: You're breathtaking!"
+excerpt: "."
```
Comment (akien-mga) [inline collections/_article/beyond-100000-you-re-breathaking.md]:
```suggestion
excerpt: "As we reach issue #100000 on GitHub, we celebrate with a little retrospective on our issue and PR growth."
```

File: collections/_article/beyond-100000-you-re-breathaking.md [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+
+---
+
+That's all for now, I was already way too long and verbose while drafting this at the last minute to (try to) snatch the 100,000th issue number ;)
+
+Aside from showing some cool numbers, I mostly want to convey that we are well aware that we have a significant backlog, though it's not as dire as it might look from the outside.
+
+To deal with it, we need better triage and review processes (which we are designing now), more volunteers involved in these processes, but also importantly [**more funding**](https://fund.godotengine.org/). Volunteer contributors do a ton of work, but many critical parts of the workflow depend on a few paid contractors, and we need to grow that group to better manage the increasing scale of the project.
+
+[^1]: I'll make a quick note that 100,000 is the combined number of issues and PRs, which share the same index system on GitHub.<br><br>At the time of writing, we've actually had 53,648 issues and 45,213 PRs created. Astute minds will notice that the sum is not 100,000, the difference comes from spam issues or PRs which have been deleted by GitHub.
```
Comment (akien-mga) [inline collections/_article/beyond-100000-you-re-breathaking.md]: I'm amazed to see this feature works in Jekyll markdown too :D

**Review by akien-mga — APPROVED** [review]
Looks good to me!

File: collections/_article/beyond-100000-you-re-breathtaking.md [hunk 5]
```diff
@@ -0,0 +1,100 @@
+---
+title: "Beyond #100000: You're breathtaking!"
+excerpt: "As we reach issue #100000 on GitHub, we celebrate with a little retrospective on our issue and PR growth."
```
Comment (JohnVeness) [inline collections/_article/beyond-100000-you-re-breathtaking.md]:
```suggestion
excerpt: "As we reach issue 100,000 on GitHub, we celebrate with a little retrospective on our issue and PR growth."
```

File: collections/_article/beyond-100000-you-re-breathtaking.md [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+categories: ["news"]
+author: "Rémi Verschelde"
+image: /storage/blog/covers/beyond-100000-you-re-breathtaking.webp
+image_caption_title: "\"You're breathtaking!\""
+image_caption_description: "Sketch of Keanu Reeves at the 2019 #XboxE3 event"
+date: 2024-12-04 19:00:00
+---
+
+<div markdown=1 class="card card-info" style="margin-top: 1em;">
+This blog post has been originally posted on [GH-100000](https://github.com/godotengine/godot/issues/100000).
```
Comment (JohnVeness) [inline collections/_article/beyond-100000-you-re-breathtaking.md]:
```suggestion
This blog post was originally posted on [GH-100000](https://github.com/godotengine/godot/issues/100000).
```

File: collections/_article/beyond-100000-you-re-breathtaking.md [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+### Pull Requests
+
+![Graph showing pull request count over time](https://github.com/user-attachments/assets/cf5027a6-4ad3-49bf-9834-d0e1cab2fdc9)
+<div markdown=1 class="small text-center">
+*Visualization from OSS Insight showing pull request count over time, [see the interactive version](https://ossinsight.io/analyze/godotengine/godot#pr-history).*
+</div>
+
+I didn't annotate this one, but a few takeaways:
+
+- Godot gets almost the same number of new pull requests per month than new issues, roughly 600 of each per month (that's 20 PRs and 20 issues per day).
```
Comment (JohnVeness) [inline collections/_article/beyond-100000-you-re-breathtaking.md]:
```suggestion
- Godot gets almost the same number of new PRs per month as new issues, roughly 600 of each per month (that's 20 PRs and 20 issues per day).
```

File: collections/_article/beyond-100000-you-re-breathtaking.md [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+---
+
+That's all for now, I was already way too long and verbose while drafting this at the last minute to (try to) snatch the 100,000th issue number ;)
+
+Aside from showing some cool numbers, I mostly want to convey that we are well aware that we have a significant backlog, though it's not as dire as it might look from the outside.
+
+To deal with it, we need better triage and review processes (which we are designing now), more volunteers involved in these processes, but also importantly [**more funding**](https://fund.godotengine.org/). Volunteer contributors do a ton of work, but many critical parts of the workflow depend on a few paid contractors, and we need to grow that group to better manage the increasing scale of the project.
+
+[^1]: I'll make a quick note that 100,000 is the combined number of issues and PRs, which share the same index system on GitHub.<br><br>At the time of writing, we've actually had 53,648 issues and 45,213 PRs created. Astute minds will notice that the sum is not 100,000, the difference comes from spam issues or PRs which have been deleted by GitHub.
```
Comment (JohnVeness) [inline collections/_article/beyond-100000-you-re-breathtaking.md]:
```suggestion
[^1]: I'll make a quick note that 100,000 is the combined number of issues and PRs, which share the same index system on GitHub.<br><br>At the time of writing, we've actually had 53,648 issues and 45,213 PRs created. Astute minds will notice that the sum is not 100,000; the difference comes from spam issues or PRs which have been deleted by GitHub.
```

File: collections/_article/beyond-100000-you-re-breathtaking.md [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+To this:
+
+![Screenshot of PVKK: Planetenverteidigungskanonenkommandant in Godot 4.3](/storage/blog/beyond-100000-you-re-breathtaking/godot-4-3.webp)
+<div markdown=1 class="small text-center">
+*Screenshot of [PVKK: Planetenverteidigungskanonenkommandant](https://store.steampowered.com/app/2956040/PVKK_Planetenverteidigungskanonenkommandant/) in Godot 4.3.*
+</div>
+
+That's no small achievement, so I encourage all contributors to take a minute to contemplate the progress that we've made together over this journey so far!
+
+Amidst the daily churn of fixing issues, reviewing PRs, making releases, etc., it's important to remind ourselves of where we are, and how we got there.
```
Comment (JohnVeness) [inline collections/_article/beyond-100000-you-re-breathtaking.md]:
```suggestion
Amidst the daily churn of fixing issues, reviewing pull requests, making releases, etc., it's important to remind ourselves of where we are, and how we got there.
```

**Comment by JohnVeness** [comment]
> > This is great! Can we add the images etc to our own hosting? I'm not sure if github user content links are entirely stable.
> 
> Done! I reduced the crap out of the gifs and the pngs also.

Looks like there's still a couple as GitHub user-attachments...

**Comment by adamscott** [comment]
> Looks like there's still a couple as GitHub user-attachments...

Thanks @JohnVeness! I fixed it.

File: collections/_article/beyond-100000-you-re-breathtaking.md (hunk 5 above)
Comment (adamscott) [reply collections/_article/beyond-100000-you-re-breathtaking.md]: I'm not sure, as it's issue #(number) 100000, isn't it? Maybe #100,000?

File: collections/_article/beyond-100000-you-re-breathtaking.md (hunk 9 above)
Comment (adamscott) [reply collections/_article/beyond-100000-you-re-breathtaking.md]: I wrapped all PR and PRs mentions with \<abbr\>. It looks like this.

Capture d’écran, le 2024-12-04 à 15 29 10

**Comment by coppolaemilio** [comment]
Thanks!

File: collections/_article/beyond-100000-you-re-breathtaking.md (hunk 5 above)
Comment (JohnVeness) [reply collections/_article/beyond-100000-you-re-breathtaking.md]: Maybe. For some reason, I just thought the # looked out of place in the excerpt, even though it's common syntax in GitHub comments etc. I don't really mind either way though.

**Pull request merged**